builder.config.json

```diff
@@ -277,29 +277,54 @@
       {
         "id": "10",
         "name": "name",
-        "display_label": "Name (Text)"
+        "display_label": "Name (Text)",
+        "data_type": "text"
       },
       {
         "id": "11",
         "name": "comment",
-        "display_label": "Comment (Text)"
+        "display_label": "Comment (Text)",
+        "data_type": "text"
       },
       {
         "id": "12",
-        "name": "start_date",
-        "display_label": "Start Date (Date)"
+        "name": "age",
+        "display_label": "Age (Number)",
+        "data_type": "number"
+      },
+      {
+        "id": "13",
+        "name": "phone_number",
+        "display_label": "Phone number (phone)",
+        "data_type": "phone"
+      },
+      {
+        "id": "14",
+        "name": "favourite_fruits",
+        "display_label": "Favourite fruits (Multiple_choice)",
```

Comment:
I assume `(Multiple_choice)` is a property name from the `subscriberPropertyFieldDataTypesMapping` mapping. Shouldn't we keep it lowercase then, as in case with `Phone number (phone)`

Reply:
You're correct, let me just use lower case.